src/modules/Host.rb

```diff
@@ -343,6 +346,22 @@ def SetModified
 
   private
 
+    # Semantic AutoYaST profile check
+    #
+    # Problems will be stored in AutoInstall.issues_list.
+    # @param [Hash] input autoyast settings
+    def check_profile_for_errors(imported_hosts)
+      # Checking for empty hostnames
+      imported_hosts.each do |ip,hosts|
```

Comment:
As commented previously, duplicate ips with the current profiles means, `hosts.size > 2` if I'm not wrong